patches/clock-0.7.2.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/System/Clock.hsc b/System/Clock.hsc
```

Comment:
Are we generally planning to have our GHC patches here rather than in our nixpkgs fork? Is there a reason for that?

Reply:
These are patches for haskell packages. We are not reusing any of the haskell packages from nixpkgs. The packages we use come straight out of the nix-tools infrastructure. Either via a defined stack snapshot or the corresponding translation of a cabal plan file.